cmd/dfget/app/root.go

```diff
@@ -64,15 +71,24 @@ func init() {
 
 // runDfget does some init operations and starts to download.
 func runDfget() error {
+	// get config from property files
+	propResults := initProperties()
+
 	// initialize logger
 	if err := initClientLog(); err != nil {
 		return err
 	}
 
-	cfg.Filter = transFilter(filter)
+	// print property load result
+	for _, propRst := range propResults {
```

Comment:
It's weird. However, I have no idea to make it elegant at present. 😢  @zcc35357949 Could you please add a TODO record here in the source code? THX.